content/user/locations/zone_definitions.md

```diff
@@ -38,6 +38,11 @@ Here are the required steps:
 - Press Insert, and select Country = "United States" and Zone = "Hawaii".  Press Insert. 
 - You're done!  You now have a new zone you can use anywhere Zone restriction is allowed, such as a shipping module's configuration. 
 
+<hr>
+
+**Note:** The procedures described in the remainder of this document
+are direct changes to your database which are best performed in phpMyAdmin, not the "Install SQL Patches" tool. 
```

Comment:
Should we add a note that if they're using any table-name "prefixes", they'll have to manually add those to each of these statements if using phpMyAdmin?

Reply:
Sure.  Adding to next checkin.